src/ds/d500/d500-device.cpp

```diff
@@ -712,10 +712,16 @@ namespace librealsense
 
     bool d500_device::check_symmetrization_enabled() const
     {
-        command cmd{ ds::MRD, 0x80000004, 0x80000008 };
+        using namespace ds;
+        command cmd(GET_HKR_CONFIG_TABLE,
+            static_cast<int>(d500_calib_location::d500_calib_ram_memory),
+            static_cast<int>(d500_calibration_table_id::stream_pipe_config_id),
+            static_cast<int>(d500_calib_type::d500_calib_dynamic));
         auto res = _hw_monitor->send(cmd);
-        uint32_t val = *reinterpret_cast<uint32_t*>(res.data());
-        return val == 1;
+        if (res.size() != sizeof(d500_stream_pipe_config_table))
+            throw std::runtime_error("Stream Config table has unexpected length");
+        auto stream_pipe_config_table = *reinterpret_cast<d500_stream_pipe_config_table*>(res.data());
```

Comment:
Should really use `check_calib< d500_stream_pipe_config_table >( res )`

Reply:
done